lib/install/template.rb

```diff
@@ -3,7 +3,13 @@
 directory "#{__dir__}/javascript", "app/javascript"
 
 puts "Copying binstubs"
-directory "#{__dir__}/bin", "bin"
+template "#{__dir__}/bin/webpack-dev-server", "bin/webpack-dev-server"
+template "#{__dir__}/bin/webpack-watcher", "bin/webpack-watcher"
+template "#{__dir__}/bin/webpack", "bin/webpack"
+if !File.exists?("bin/yarn")
```

Comment:
Since `File.exists?` deprecated in Ruby 2.1, it is better to use `File.exist?`.
Ref: https://github.com/ruby/ruby/blob/v2_1_2/file.c#L1413